packages/core/utils/lib/content-types.js

```diff
@@ -99,8 +99,8 @@ const getPrivateAttributes = (model = {}) => {
   );
 };
 
-const isPrivateAttribute = (model = {}, attributeName) => {
-  return model && model.privateAttributes && model.privateAttributes.includes(attributeName);
+const isPrivateAttribute = (model, attributeName) => {
+  return model?.privateAttributes?.includes(attributeName) || false;
```

Comment:
Just a small note, but by only referring to privateAttributes here, it won't find the component's private attributes